src/shared/containers/DashboardListContainer.tsx

```diff
@@ -29,22 +28,23 @@ const DashboardList: React.FunctionComponent<DashboardListProps> = ({
     const dashboard = dashboardResources.find(d => d['@id'] === id);
     setSelectedDashboard(dashboard);
   };
-  useAsyncEffect(async () => {
-    const dashboardList: Resource[] = [];
-    for (let i = 0; i < dashboards.length; i += 1) {
-      try {
-        const dashboard = (await nexus.Resource.get(
+  React.useEffect(() => {
+    Promise.all(
+      dashboards.map(dashboardObject => {
+        return nexus.Resource.get(
           orgLabel,
           projectLabel,
-          encodeURIComponent(dashboards[i].dashboard)
-        )) as Resource;
-        dashboardList.push(dashboard);
-      } catch (error) {
-        // TODO: display an error to the user
-      }
-    }
-    setDashboardResources(dashboardList);
-    setSelectedDashboard(dashboardList[0]);
+          encodeURIComponent(dashboardObject.dashboard)
+        );
+      })
+    )
+      .then(values => {
+        setDashboardResources(values);
+        setSelectedDashboard(values[0]);
+      })
+      .catch(e => {
+        // Fail silently.
```

Comment:
we can use andt notification API here to display an error to a user, for example something like this:
```
        notification.error({
          message: `An error occurred`,
          description: error.message,
        });
```

Reply:
It was decided earlier that we are going to come back to this later. We have not decided what error to display to the user yet.